app/controllers/sessions_controller.rb

```diff
@@ -16,8 +18,12 @@ def create
     session[:services] = services
     session[:uid] = auth_hash.uid
     persist_services_from_session if session[:services].present?
+    session[:admin] = readonly_admin
+    session[:admin_users] = admin_users
 
-    redirect_to employees_path
+
+    return if enforce_admin_permissions
```

Comment:
if I'm logged in as a non admin user, and I click sign in on http://localhost:3004/eligible, I am still taken to the right page, but the "You don't have permission to access the admin portal." notice is a little disconcerting!

![image](https://github.com/wearefuturegov/tell-us-who-you-employ/assets/649148/ebb56de0-6d18-4028-9fbd-a47b652ac06b)

Reply:
Also if I log out of Outpost and even if I click ‘finish and send’ I'm still able to see the `/employees` page